ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -548,7 +574,8 @@
       },
       {
         "UserRoles": [
-          "caseworker-civil-admin"
+          "caseworker-civil-admin",
+          "caseworker-civil-judge"
```

Comment:
I dont think a judge would need the ability to do this, please confirm this use case with the business

Reply:
I have changed the access to read only for now whilst pending business confirmation